server/views/cms/rest.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from flask import jsonify, request
+import requests
+import html
+
+from server import app
+from server.util.request import api_error_handler
+
+logger = logging.getLogger(__name__)
+BASE_URL = app.config['PAYLOAD_API_URL']
+API_KEY = app.config['PAYLOAD_API_KEY']
+
+
+@app.route('/api/cms/fetch-page-content', methods=['GET'])
+@api_error_handler
+def api_fetch_page_content():
+    application_name = html.escape(request.headers.get('cs-app') or '')
+    url = f"{BASE_URL}/{application_name}-pages"
+    headers = dict(request.headers)
+    headers['Authorization'] = f"users API-Key {API_KEY}"
```

Comment:
Why do we need `users` here?

Reply:
This is needed by Payload for [Rest API requests authentication using an API key](https://payloadcms.com/docs/authentication/api-keys#http-authentication)

Reply:
... very **_unusual_** design decision on their part; the key should be enough to confirm if it's for a user or not.

Reply:
I think they kept it this way because we can enable API keys on any collection thus they need the slug to be able to tell which collection we are authenticating against